commerce-ui/app/globals.css

```
@@ -0,0 +1,970 @@
+  --surface: #fffdf8;
+  --surface-muted: #efede6;
+  --muted: #635f57;
+  --border: #d9d5c8;
+  --border-strong: #b9b19f;
+  --accent: #355f4e;
+  --accent-foreground: #ffffff;
+  --focus: #8c5f2f;
+  --shadow-soft: 0 18px 48px rgba(23, 21, 17, 0.08);
+}
+
+@theme inline {
```

Stylelint (17.13.0)

[error] Unexpected unknown at-rule "`@theme`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)